src/oracle/config.rs

```diff
@@ -0,0 +1,36 @@
+use std::collections::HashMap;
+use std::env;
+
+use super::constants::*;
+
+/// Config carries all the configuration for Oracle services.
+#[derive(Clone, Default)]
+#[cfg_attr(test, derive(Debug))]
+pub struct Config {
+    /// `private_key` will be loaded from
+    ///
+    /// - this field if it's `is_some`
+    /// - env value: [`ORACLE_CLOUD_PRIVATE_KEY`]
```

Comment:
Does oracle's own sdk use those env?

Reply:
it seems not, should we follow oracle's practice by providing the `~/.oci/config` file? that's how oci SDK works.